src/jl_uv.c

```diff
@@ -49,6 +49,16 @@ extern "C" {
 
 static uv_async_t signal_async;
 
+#ifdef _OS_WINDOWS_
+// uv_async_t is buggy on windows. Initializing one breaks the sysimg build.
```

Comment:
You have you force aborting the libuv main loop. (pressing `Ctrl-C` multiple times instead of once). Although when I tested on windows neither the new version or the old version seems to work reliably on windows....